torch/csrc/dynamo/eval_frame.c

```diff
@@ -483,7 +485,8 @@ inline static PyObject* eval_custom_code(
     PyCodeObject* code,
     int throw_flag,
     int free_vars_copied) {
-  _PytorchRecordFunctionState* rf = _pytorch_record_function_enter("Torch-Compiled Region");
```

Comment:
What I want here, is I want a `const char*` passed in with the name of the compile region to this function, and then we use the old `_pytorch_record_function_enter`. Let me check the call sites to ensure you can do this...